### DISCUSSION TITLE
Compilation error

### DISCUSSION THREAD

**Discussion by VladSar21** [FYSETC/FYSETC-E4#15] [Q&A]
I am planning to connect an HMI DWIN display to the E4 via Bluetooth. First, I want to tune the mechanics of the Ultimaker 2. An error occurs during compilation. What could be the mistake?
![Screenshot_20210502_221823](https://user-images.githubusercontent.com/83567008/116841210-8ca0a500-abe9-11eb-874f-764c4e5a6d89.png)

**Comment by VladSar21** [comment]
OK. The distribution was missing the disk_io_driver.h file

**Comment by Juri4** [comment]
Exactly the same error

**Comment by VladSar21** [comment]
I can send the working configuration to the e-mail, I no longer remember what I changed, the only thing in my configuration is the connection with the DWIN display via the serial port.

**Comment by Juri4** [comment]
Hello!
I will be very grateful if you send it.
Thank you in advance.

**Reply by VladSar21** [reply]
https://cloud.mail.ru/public/oaQN/pS3Zx4hdS/
